flutter/shell/platform/tizen/channels/platform_channel.h

```diff
@@ -49,10 +54,12 @@ class PlatformChannel {
   // A reference to the native view managed by FlutterTizenView.
   TizenViewBase* view_ = nullptr;
 
+#ifdef CLIPBOARD_SUPPORT
+  TizenClipboard* tizen_clipboard_;
```

Comment:
You can use a `std::unique_ptr`.

You have to initialize the variable if you really want to use a raw pointer.

Reply:
oh i will update it

Reply:
done